modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSCustomView.macro

```diff
@@ -52,27 +82,11 @@ definition {
 	}
 
 	macro saveCustomView {
-		Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
-
-		Click(
-			key_optionName = "Save View",
-			locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
-	}
-
-	macro saveViewAs {
-		Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
+		Click(locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
 
 		Click(
-			key_itemName = "Save View As",
+			key_itemName = "Save View",
 			locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
-
-		Type(
-			locator1 = "FrontendDataSet#INPUT_CUSTOM_VIEW_NAME",
-			value1 = "${key_nameView}");
-
-		Click(
-			key_name = "Save",
-			locator1 = "Button#GENERAL_BUTTON_MODAL_FOOTER");
```

Comment:
We may want to keep saveViewAs. Why was it removed?

Reply:
There was two macro doing the same function.